packages/flutter/lib/src/widgets/image.dart

```diff
@@ -822,6 +835,38 @@ class Image extends StatefulWidget {
   /// {@animation 400 400 https://flutter.github.io/assets-for-api-docs/assets/widgets/loading_progress_image.mp4}
   final ImageLoadingBuilder loadingBuilder;
 
+  /// A builder function that is called if an error occurs during image loading.
+  ///
+  /// If this builder is not provided, any exceptions will be reporetd to
+  /// [FlutterError.onError]. If it is provided, the caller should either handle
```

Comment:
This should mention that handling means returning a replacement widget that will render instead of the image.

Reply:
Done